Odata-docs/TOC.yml

```diff
@@ -79,6 +79,14 @@
                 href: /odata/webapi-8/fundamentals/ref-routing
           - name: Actions and Functions
             href: /odata/webapi-8/fundamentals/actions-functions
+          - name: Pagination
+            items:
+              - name: Pagination overview
```

Comment:
nit: For consistency in the docs, consider sticking to either paging or pagination instead of both